nautobot/extras/models/models.py

```diff
@@ -448,6 +450,49 @@ def clean(self):
             self.file_extension = self.file_extension[1:]
 
 
+#
+# External integrations
+#
+
+
+class ExternalIntegration(PrimaryModel):
+    """Model for tracking integrations with external applications."""
+
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
MySQL is OK (doesn't complain) with a unique field of size 255, right?

Reply:
255 is the maximum allowed: https://docs.djangoproject.com/en/4.2/ref/databases/#mysql-character-fields